README.md

```diff
@@ -80,7 +80,7 @@ See the [Cassandra error handling done right blog](https://www.datastax.com/blog
 * [FAQ]
 
 [API docs]: https://docs.datastax.com/en/drivers/java/4.17
-[JIRA]: https://datastax-oss.atlassian.net/browse/JAVA
+[JIRA]: https://issues.apache.org/jira/projects/CASSANDRA/issues/
```

Comment:
Mentioned in separate conversation by @ekaterinadimitrova2: we should mention the "Client/java-driver" component we've been using for Java driver tickets here.

Reply:
I'm kinda wondering if there isn't some way we could pre-fill in that component by customizing this link a bit, maybe with some query params or extra URL args...

Reply:
```suggestion
[JIRA]: https://issues.apache.org/jira/issues/?jql=project%20%3D%20CASSANDRA%20AND%20component%20%3D%20%22Client%2Fjava-driver%22%20ORDER%20BY%20key%20DESC
```
? 

there's no way to assign the component when creating a new issue by url or template…

Reply:
Ah, that's an excellent idea @michaelsembwever !  I'll also add some quick text here to make it clear that "Client/java-driver" is selected.